packages/mui-material-next/src/InputBase/utils.ts

```diff
@@ -0,0 +1,37 @@
+// Supports determination of isControlled().
+// Controlled input accepts its current value as a prop.
+//
+// @see https://facebook.github.io/react/docs/forms.html#controlled-components
+// @param value
+// @returns {boolean} true if string (including '') or number (including zero)
+
+// TODO: drop this? it looks unused
```

Comment:
If unused we should drop it

Reply:
Actually my comment is wrong, I thought this was a duplicate of something that could be imported from Base but actually it isn't 🙈